javascript/ql/src/semmle/javascript/frameworks/NodeJSLib.qll

```diff
@@ -351,11 +351,73 @@ module NodeJSLib {
 
     override DataFlow::Node getAPathArgument() {
       exists (int i | fsFileParam(methodName, i) |
-        result = getArgument(i)
+        result = getArgument(i))
+    }
+  }
+
+  /**
+   * A call to write corresponds to a pattern where file stream is open first, followed by 'read' or 'write' call
```

Comment:
The class doesn't seem to check for a call to `open` or similar. Is that intentional?

Reply:
I've split this class in two and now I can check for createReadStream and createWriteStream methods